tests/test_wheel.py

```diff
@@ -70,19 +72,21 @@ def test_read_valid(example_wheel):
 
 def test_read_non_existent_wheel_file_name():
     """Raise an exception when wheel file doesn't exist."""
-    file_name = "/foo/bar/baz.whl"
+    file_name = str(os.path.abspath(pathlib.Path("/foo/bar/baz.whl")))
     with pytest.raises(
-        exceptions.InvalidDistribution, match=f"No such file: {file_name}"
+        exceptions.InvalidDistribution, match=re.escape(f"No such file: {file_name}")
```

Comment:
I'm curious why `re.escape` was added. The tests pass locally without it. Is it a Windows issue?

Reply:
Yes, it's a windows issue with having to escape the backslash in Windows paths